src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -5,84 +5,98 @@ suppressMessages({
   library(patchmax)
   library(ggplot2)
   library(ggnewscale)
+  # TODO: remove below
+  library(tidyverse)
```

Comment:
Can this be removed here if it's included in the clustering R file?

Reply:
I am using this for debugging purposes (`write_rds()` function) and can remove it. Ideally I'd also like to restrict the clustering function to just `dplyr` instead of the whole `tidyverse`. Will test that now.

Reply:
I see.

Is it possible to document your response in comments in the code?

Reply:
I ended up sticking with tidyverse because patchmax requires it anyway